engine/src/player-platform.cpp

```diff
@@ -181,12 +181,8 @@ class MCPlayerVolumePopup: public MCStack
         t_volume_well = getVolumeBarPartRect(dirty, kMCPlayerControllerPartVolumeWell);
         
         MCGBitmapEffects t_effects;
-        t_effects . has_drop_shadow = false;
-        t_effects . has_outer_glow = false;
-        t_effects . has_inner_glow = false;
-        t_effects . has_inner_shadow = true;
-        t_effects . has_color_overlay = false;
-        
+        MCMemoryClear(t_effects);
```

Comment:
@livecodeian this patch changes the inited value of `t_effects . has_inner_shadow = true;` is that right?

Reply:
@livecodeian actually `MCGBitmapEffects` has default member initializers so setting all members to `false` so both `MCMemoryClear` and all the setting to `false` everywhere should be unnecessary.

Reply:
The default values aren't used unless a constructor is called - though that might be a better way of doing things than MCMemoryClear(), i.e. 
```
MCGBitmapEffects t_effects = MCGBitmapEffects();
t_effects . has_inner_shadow = true;
```